packages/paste-core/components/form/src/RadioGroup.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {InlineControlGroup, InlineControlGroupProps} from './shared/InlineControlGroup';
+
+export interface RadioGroupProps extends InlineControlGroupProps {
+  onChange: (value: string) => void;
+  name?: string;
+}
+
+const RadioGroup: React.FC<RadioGroupProps> = ({onChange, children, name, value, disabled, ...props}) => {
+  return (
+    <InlineControlGroup {...props} disabled={disabled} name={name}>
+      {React.Children.map(children, child => {
+        return React.isValidElement(child)
+          ? React.cloneElement(child, {
+              name,
+              onChange: (event: React.ChangeEvent<HTMLInputElement>): void => {
+                if (onChange != null) {
+                  onChange(event.target.value);
+                }
+              },
+              checked: child.props.value === value,
+              disabled,
```

Comment:
@SiTaggart do we need to pass `hasError` and `required` down to the individual radios and checkboxes at the group level? Similar to what we're doing with `disabled`?

Reply:
Potentially for error, maybe? If the group is in an error state, all the options are errored. 

As for required, I don't think so. Just becuase a group of checkboxes has a requirement, doesn't mean _all_ checkboxes should be checked, it might only be one. And it's not really applicable for radios either, as you can only pick one, meaning all the others are unchecked but still required individually.

I actually don't even think it's a thing that should be exposed for a single radio. If it is on their interface we should probably remove it.

Reply:
Just pushed the fix for passing `hasError`. We were never exposing `required`, so we're good there.